tests/integration/components/copy-button-test.js

```diff
@@ -74,6 +95,92 @@ test('error action fires', function(assert) {
   this.$('button').click();
 });
 
+test('error action fires with delegateClickEvent: false', function(assert) {
+  assert.expect(1);
+
+  this.on('error', () => {
+    assert.ok(true, 'error action successfully called');
+  });
+
+  this.render(hbs`
+    {{#copy-button
+      clipboardText='text'
+      delegateClickEvent=false
+      success='success'
+      error='error'
+    }}
+      Click To Copy
+    {{/copy-button}}
+  `);
+
+  /*
+   * Can only directly test error case here b/c browsers do not allow simulated
+   * clicks for `execCommand('copy')`. See test-helpers to test action integration.
+   */
+  this.$('button').click();
+});
+
```

Comment:
These two tests below assert the setup logic in didInsertElement and will fail.  The tests above two are just happy path, smoke tests.